test/verify/machineslib.py

```diff
@@ -1296,7 +1296,8 @@ def allowBugErrors(location, original_exception):
                 error_message = b.text(location)
 
                 if "CPU is incompatible with host CPU" not in error_message and \
-                                "unsupported configuration: CPU mode" not in error_message:
+                                "unsupported configuration: CPU mode" not in error_message and \
+                                "Could not allocate dynamic translator buffer" not in error_message: # allow to run with kvm disabled
```

Comment:
I had the impression that this error message happened in the actual dialog box, it wasn't jusj a test-only error. This change now ignores this, but does that whole test still actually make sense now? I. e. on a machine without nested kvm, can the test still create some VMs (in slow emulation mode) to make sure that our code works?

Reply:
It appears to me that this error fails `{create,install}-machine.sh`, and thus in such an environment we can never get a successful VM creation?

Reply:
Oh, you 're right. And I just catch the Exception and the test will never actually execute properly, just silently pass.
I guess I 'll close the PR, doesn't make any sense.